DanielN88/modern-feed
# Set up app

This PR sets up the initial boilerplate code for the app. Installs react, cypress, router, and sets up app. 

Where to start?
At the App file. 

How to test?
In the project directory, you can run:

npm start
Runs the app in the development mode.
Open http://localhost:3000/ to view it in your browser.
The page will reload when you make changes.

Type of Change:

-[] Bug fix (non-breaking change which fixes an issue)
-[] Refactor (code is working and without bugs, formatting has been streamlined)
-[x] New feature (non-breaking change which adds functionality)
-[] Breaking change (fix or feature that would cause existing functionality to not work as expected)

 Have you used Dev Tools/ ran code in the console?

-[] Have you ran the Cypress testing suite?
Checklist:
-[x] My code follows the style guidelines of this project.
-[x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
-[x] I have performed a self-review of my own code.
-[] I have added reviewers.
